### ISSUE TITLE
Wrong message for excluded rules

### ISSUE THREAD

**Issue by BernieWhite** [microsoft/PSRule#1493]
**Description of the issue**

When excluded resources are reported an incorrect message is displayed

**Error output**

```text
Suppression group 'PSRule.Rules.Azure\Azure.KeyVault.Logs' has expired and will be ignored.
```

**Module in use and version:**

- Module: PSRule
- Version: **2.8.0**
